QuizApp/QuizApp/Client/Quiz/QuizClient.swift

```diff
@@ -2,6 +2,8 @@ protocol QuizClientProtocol {
 
     func getQuizes(for category: QuizCategoryClientModel) async throws -> [QuizResponseClientModel]
 
+    func getQuizes() async throws -> [QuizResponseClientModel]
```

Comment:
Functions that return a value and have no arguments should be computed properties.